data/human/events.txt

```diff
@@ -2951,3 +2954,20 @@ mission "Geminus Rebuilt"
 	on offer
 		event "geminus rebuilt" 90
 		fail
+
+event "martini rebuilt"
+	planet "Martini"
+		description `Martini is a world of exotic beaches and perfect weather. The Republic's central bank and the galactic stock exchange were previously headquartered here, before a terrorist attack destroyed them and a large portion of the old capital city as well.`
+		description `	The capital; which will be rebuilt when the radiation clears away; is currently sealed off from public access, an unnecessary reminder of the day when millions of people died at the hands of Syndicate executives so that a recession could be averted. It is strange to hear the 'evil of money' now espoused by a population that once lived and thrived off that very object.`
```

Comment:
Assuming, for example, you go down the # route, you never find out that the Syndicate were responsible (At least as of right now) so how would that work?

Reply:
True, I'll change this.